src/controllers/authentication.ts

```diff
@@ -0,0 +1,30 @@
+import express from 'express'
+import User from '../database/models/userModel'
+import Token from '../database/models/token.model'
+
+const verifyEmail = async (req: express.Request, res: express.Response) => {
+  try {
+    const { token } = req.params
```

Comment:
Handle a case where the token param does not exist?

Reply:
you can also use the validation schema to make usre that use provide a valid param